include/shark/Algorithms/DirectSearch/CMA.h

```diff
@@ -204,6 +204,20 @@ class CMA : public AbstractSingleObjectiveOptimizer<RealVector >
 		return m_numEvaluations;
 	}
 
+	///\breif Return the population form the previous step
+	std::vector<SolutionType> population() const {
+		return m_population;
+	}
+
+	///\brief Returns the best solution of the previous step
+	SolutionType best() const {
```

Comment:
this is already implemented in the optimizer base class

Reply:
Alright, I'll remove it in a PR update.